sidebars.js

```diff
@@ -1,295 +1,301 @@
-        label: "Java Development Patterns",
-        items: [
-          "developer/java/patterns/java_app_layering",
-          "developer/java/patterns/java_mapstruct_mapping",
-          "developer/java/patterns/java_swagger_annotations",
-        ],
-      },
-    ],
-    Workloads: [
-      "workloads/workloads",
-      {
-        type: "category",
-        label: "Common",
+        label: "Development Support",
         items: [
+          "developer/developer_patterns_readme",
           {
             type: "category",
-            label: "Java Spring Boot REST API",
+            label: "Java Development Patterns",
             items: [
-              "workloads/common/backend/java/intro_java",
-              "workloads/common/backend/java/scaffolding_java",
-              "workloads/common/backend/java/ide_java",
-              "workloads/common/backend/java/maven_spring_profiles",
-              {
-                type: "category",
-                label: "Architecture",
-                items: [
-                  "workloads/common/backend/java/architecture/maven_modules_java",
-                  {
-                    type: "category",
-                    label: "Modules",
-                    items: [
-                      "workloads/common/backend/java/architecture/dependency_parent_java",
-                      "workloads/common/backend/java/architecture/dependency_commons_java",
-                      "workloads/common/backend/java/architecture/dependency_api_java",
-                      "workloads/common/backend/java/architecture/dependency_cqrs_java",
-                      "workloads/common/backend/java/architecture/dependency_messaging_java",
-                    ],
-                  },
-                  {
-                    type: "category",
-                    label: "Web API Architecture",
-                    items: [
-                      "workloads/common/backend/java/architecture/web_api/architecture_java",
-                      "workloads/common/backend/java/architecture/web_api/structure_java",
-                    ],
-                  },
-                  {
-                    type: "category",
-                    label: "Web API with CQRS and Events Architecture",
-                    items: [
-                      "workloads/common/backend/java/architecture/java_cqrs/architecture_java_cqrs",
-                      "workloads/common/backend/java/architecture/java_cqrs/structure_java_cqrs",
-                    ],
-                  },
-                ],
-              },
+              "developer/java/patterns/java_app_layering",
+              "developer/java/patterns/java_mapstruct_mapping",
+              "developer/java/patterns/java_swagger_annotations",
             ],
           },
-        ],
+        ]
       },
       {
         type: "category",
-        label: "Azure",
+        label: "Ensono Stacks CLI",
+        items: [
+          "stackscli/about"
+          // "stackscli/usage",

```

Comment:
maybe best to delete these if they are not being used

Reply:
updated